packages/chrome/src/styled/sheet/StyledSheet.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'chrome.sheet';
+
+export const SHEET_SIZE = 380;
+
+interface IStyledSheetProps {
+  placement?: 'start' | 'end';
+  isOpen?: boolean;
+  isAnimated?: boolean;
+  size?: number;
+}
+
+const sheetBorderStyle = ({
+  theme,
+  placement,
+  isOpen
+}: IStyledSheetProps & ThemeProps<DefaultTheme>) => {
+  const borderColor = isOpen ? getColor('neutralHue', 300, theme) : 'transparent';
+  let borderSide = '';
+
+  // todo: clean up
+  if (placement === 'end') {
+    if (theme.rtl) {
+      borderSide = '-right';
+    } else {
+      borderSide = '-left';
+    }
+  } else if (placement === 'start') {
+    if (theme.rtl) {
+      borderSide = '-left';
+    } else {
+      borderSide = '-right';
+    }
+  }
+
+  return `border${borderSide}: ${theme.borders.sm} ${borderColor};`;
+};
+
+export const StyledSheet = styled.aside.attrs({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledSheetProps>`
+  display: flex;
+  position: relative;
+  order: 1;
+  ${props =>
+    css`
+      transition: ${props.isAnimated ? 'width 0.5s ease-in-out, border 0.5s ease-in-out' : 'none'};
```

Comment:
The `border` should not be animated. Either it is visible on `isOpen` or transparent. Also, some UAs are not able to properly animate from `transparent`. Also, this speed feels very sluggish for Garden. Compare with DrawerModal that animates at 0.25s.

Reply:
This construct with extra `css` is not necessary. Use:

```
transition: ${props => props.isAnimated && 'width 0.25s ease-in-out'};
```

Reply:
Will update the speed of the transition and will remove border transition.